Plutarch/Rational.hs

```diff
@@ -192,48 +217,36 @@ instance PFractional PRational where
     phoistAcyclic $
       plam $ \x ->
         pmatch x $ \(PRational xn xd) ->
-          pfailOnZero # xn # pcon (PRational xd xn)
+          pcon $ PRational (pto xd) $ ptryPositive # xn
 
   x' #/ y' =
```

Comment:
We can implement this in terms of `#*`. Add a TODO please.

Reply:
added